requirements-python3/dev-requirements.txt

```diff
@@ -65,7 +65,7 @@ django-transfer==0.4
 django-two-factor-auth==1.6.2
 django-user-agents==0.3.2
 django-websocket-redis==0.5.2
-django==1.11.22
+django==2.0.13
```

Comment:
I think you'll likely want to pin this better in requirments.in. I can't imagine we'd want to upgrade all of these and django at the same time (unless minor version)

I had been running this once a week for a while, but been putting it off until after the deprecation of py2

Reply:
nevermind i guess. I see from your branch that you know how reckless you're being. carry on